### ISSUE TITLE
Can't seem to get autofill to work???

### ISSUE THREAD

**Issue by Frazzle1** [PhilippC/keepass2android#1668]
Keepass 2 android autofill service plugin Problem
steps done to install

1. Download the keepass plugin file (kp2a.plugin.AutoFill-1.02.apk) at the link below and move into the downloads folder on the phone.
https://github.com/PhilippC/kp2a_accservice_autofill/releases/
2. First allow the file manager app to install unknown apps by going to Settings -->Apps and Notifications --> then select the file manager app --> Advanced --> Install unknown apps -->Set Allow from this service button to on. Then install the plugin by opening the Android file manager app, going to the download folder and tapping on the plug in APK file. 
3. Go to Settings --> Accessibility --> KP2A AutoFillPlugin --> set Use KP2A AutoFillPlugin button switch to on --> Select Allow for allowing KP2A AutoFillPlugin to have full control of your device.
4. Set AutoFillPlugin shortcut ( hold volume keys) to on. Don't understand the significance of this option?
5. Go to Settings --> System --> Languages and Input --> Advanced --> Auto-fill service and select Keepass2Android
6. Open Keepass2Android and go into it's Settings --> Plug-ins -->Enable AutoFill-Plugin (Accessibility Service) 1.02

After all this The auto fill function in my Google Chrome browser fails to find the correct username and password entries in my keepass2android database. So what am I doing wrong here! This is frustration to the nth degree! I have a pixel 3 phone running the latest Android 11 operating system version. I thought I read somewhere that this plug in does not work with the google chrome browser app? Is this the problem???

**Comment by PhilippC** [comment]
I am no longer supporting this plugin as it uses the outdated "accessibility service". Android 8 introduced an Autofill API which is supported by the app without needing a plugin.

**Comment by Frazzle1** [comment]
Thanks Phillip I uninstalled the plugin and got the autofill working with Keepass2android on my phone now. Nice feature, so convenient! Sure beats looking up manually in Keepass every entry I want to log into. Only 1 setting I had to change in my android phone settings. Settings --> System --> Languages and Input --> Advanced --> Auto-fill Service, then select Keepass2Android. Thanks again.

⁣Sent from BlueMail ​
